ktor-client/ktor-client-core/posix/src/io/ktor/client/engine/Loader.kt

```diff
@@ -7,15 +7,15 @@ package io.ktor.client.engine
 import io.ktor.util.*
 import kotlin.native.concurrent.*
 
-private typealias T = HttpClientEngineFactory<HttpClientEngineConfig>
+private typealias T = Pair<String, HttpClientEngineFactory<HttpClientEngineConfig>>
```

Comment:
Consider overriding `toString` in `HttpClientEngineFactory`

Reply:
Done.